WireAnalytics/Sources/WireAnalytics/Service/AnalyticsEventTracker.swift

```diff
@@ -16,16 +16,16 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import WireAnalytics
+import Foundation
 
-final class MockAnalyticsManagerProviding: AnalyticsManagerProviding {
+// sourcery: AutoMockable
+/// An object that tracks analytic events.
+public protocol AnalyticsEventTracker: AnyObject {
```

Comment:
The user session doesn't need to know about the full `AnalyticsService` so we create a protocol only for the track event method to hide the rest of the `AnalyticsService`.